src/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -335,6 +335,7 @@ public sealed partial class TypeBuilder : System.Type //TYPEINFO: System.Reflect
         public override System.Reflection.GenericParameterAttributes GenericParameterAttributes { get { throw null; } }
         public override int GenericParameterPosition { get { throw null; } }
         public override System.Guid GUID { get { throw null; } }
+        public override bool IsByRefLike { get { throw null; } }
```

Comment:
Did this one get approved? I do'nt see it in the issue.

Reply:
Also I don't see tests for it?

Reply:
Members introduced on Type do get overridden on public Type-derived classes as a matter of course - we tend not to call that out explicitly.

Reply:
Fine. Do we need tests for the ones in xxxBuilder?

Reply:
They'll just return false on Builder types so tests aren't too interesting. Anyway, I can't add them yet - CoreCLR doesn't overrride these on Builder types yet. I've opened an issue to take care of that little detail (https://github.com/dotnet/coreclr/issues/13662)